OpenChange/RTFHandler.m

```diff
@@ -293,6 +294,16 @@ - (void) dealloc
   [super dealloc];
 }
 
+- (NSString *) description
+{
+  NSString *description;
+  description = [NSString stringWithFormat:
+                          @"%u name=%@ family=%@family charset=%@ pitch=%u",                
```

Comment:
This change conflicts with the #209 changeset. I thing the other one is the correct one, could you fix this svp?